api/src/modules/indicator-records/indicator-records.service.ts

```diff
@@ -110,83 +109,82 @@ export class IndicatorRecordsService extends AppBaseService<
     const indicatorRecords: IndicatorRecord[] = [];
 
     await Promise.all(
-      indicators.map(
-        async (indicator: Indicator): Promise<void> => {
-          const indicatorH3Table:
-            | H3Data
-            | undefined = await this.h3DataService.findH3ByIndicatorId(
-            indicator.id,
-          );
+      indicators.map(async (indicator: Indicator): Promise<void> => {
+        const indicatorH3Table: H3Data | undefined =
+          await this.h3DataService.findH3ByIndicatorId(indicator.id);
 
-          if (!indicatorH3Table) {
-            throw new Error(
-              'Cannot calculate impact for sourcing record - missing indicator h3 data',
-            );
-          }
+        if (!indicatorH3Table) {
+          throw new Error(
+            'Cannot calculate impact for sourcing record - missing indicator h3 data',
+          );
+        }
 
-          let impactValue: number | null = null;
+        let impactValue: number | null = null;
 
-          switch (indicator.nameCode) {
-            case 'UWU_T':
-              impactValue = await this.h3DataService.getWaterRiskIndicatorRecordValue(
+        switch (indicator.nameCode) {
+          case 'UWU_T':
```

Comment:
Hack alert: I suggest to use the indicator type enum both for readability and to avoid potential bugs due to magic string.
Not actually related to this PR, but reminding myself of a clear example of things to improve while cleaning

Reply:
+1